packages/eslint-plugin/package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-plugin-deprecation": "^1.2.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^23.13.2",
+    "eslint-plugin-jsdoc": "^36.0.6",
```

Comment:
pls add this plugin to root package.json with pinned version as well under devDeps 
-> single version policy